packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -713,6 +727,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             },
         );
 
+        if (this.context.baseSnapshot) {
```

Comment:
I am not sure what this does. Can you please explain a little and also add a comment?